test/bullet_train/routes_test.rb

```diff
@@ -1,7 +1,187 @@
+     projects_deliverable GET    /projects/deliverables/:id(.:format)      projects/deliverables#show
+                          PATCH  /projects/deliverables/:id(.:format)      projects/deliverables#update
+                          PUT    /projects/deliverables/:id(.:format)      projects/deliverables#update
+                          DELETE /projects/deliverables/:id(.:format)      projects/deliverables#destroy
+    ROUTES
+  end
+
+  test "nested model routes" do
+    model "Project" do
+      model "Projects::Deliverable"
+    end
+
+    assert_formatted_routes <<~ROUTES
+             project_deliverables GET    /projects/:project_id/deliverables(.:format)                   projects/deliverables#index
+                                  POST   /projects/:project_id/deliverables(.:format)                   projects/deliverables#create
+          new_project_deliverable GET    /projects/:project_id/deliverables/new(.:format)               projects/deliverables#new
+edit_project_projects_deliverable GET    /projects/:project_id/projects/deliverables/:id/edit(.:format) projects/deliverables#edit
+     project_projects_deliverable GET    /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#show
+                                  PATCH  /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#update
+                                  PUT    /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#update
+                                  DELETE /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#destroy
+                         projects GET    /projects(.:format)                                            projects#index
+                                  POST   /projects(.:format)                                            projects#create
+                      new_project GET    /projects/new(.:format)                                        projects#new
+                     edit_project GET    /projects/:id/edit(.:format)                                   projects#edit
+                          project GET    /projects/:id(.:format)                                        projects#show
+                                  PATCH  /projects/:id(.:format)                                        projects#update
+                                  PUT    /projects/:id(.:format)                                        projects#update
+                                  DELETE /projects/:id(.:format)                                        projects#destroy
+    ROUTES
+
+    assert_routing_equal_to do
+      collection_actions = [:index, :new, :create]
+
+      resources :projects do
+        scope module: 'projects' do
+          resources :deliverables, only: collection_actions
+        end
+      end
+
+      namespace :projects do
+        resources :deliverables, except: collection_actions
+      end
+    end
+  end
```

Comment:
It looks like there's a mismatch between the README example here and what's generated from `namespace` + `resources`.

```
Failure:
BulletTrain::RoutesTest#test_nested_model_routes [/Users/kasperhansen/code/bullet-train-co/bullet_train-routes/test/bullet_train/routes_test.rb:71]:
Expected routes to be equal, but found these differences.

These are the expected routes:
  edit_projects_deliverable GET    /projects/deliverables/:id/edit(.:format)        projects/deliverables#edit
     projects_deliverable GET    /projects/deliverables/:id(.:format)             projects/deliverables#show
                          PATCH  /projects/deliverables/:id(.:format)             projects/deliverables#update
                          PUT    /projects/deliverables/:id(.:format)             projects/deliverables#update
                          DELETE /projects/deliverables/:id(.:format)             projects/deliverables#destroy

These were the actual routes:
  edit_project_projects_deliverable GET    /projects/:project_id/projects/deliverables/:id/edit(.:format) projects/deliverables#edit
     project_projects_deliverable GET    /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#show
                                  PATCH  /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#update
                                  PUT    /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#update
                                  DELETE /projects/:project_id/projects/deliverables/:id(.:format)      projects/deliverables#destroy

The full expected route set is:
                   Prefix Verb   URI Pattern                                      Controller#Action
     project_deliverables GET    /projects/:project_id/deliverables(.:format)     projects/deliverables#index
                          POST   /projects/:project_id/deliverables(.:format)     projects/deliverables#create
  new_project_deliverable GET    /projects/:project_id/deliverables/new(.:format) projects/deliverables#new
                 projects GET    /projects(.:format)                              projects#index
                          POST   /projects(.:format)                              projects#create
              new_project GET    /projects/new(.:format)                          projects#new
             edit_project GET    /projects/:id/edit(.:format)                     projects#edit
                  project GET    /projects/:id(.:format)                          projects#show
                          PATCH  /projects/:id(.:format)                          projects#update
                          PUT    /projects/:id(.:format)                          projects#update
                          DELETE /projects/:id(.:format)                          projects#destroy
edit_projects_deliverable GET    /projects/deliverables/:id/edit(.:format)        projects/deliverables#edit
     projects_deliverable GET    /projects/deliverables/:id(.:format)             projects/deliverables#show
                          PATCH  /projects/deliverables/:id(.:format)             projects/deliverables#update
                          PUT    /projects/deliverables/:id(.:format)             projects/deliverables#update
                          DELETE /projects/deliverables/:id(.:format)             projects/deliverables#destroy
```